### PR TITLE
Enable result metadata for Cloudformation and quickstart

### PR DESCRIPTION
**Pull request by shruti0085**
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description

Enabled metadata `Result` for CloudFormation open stack metric and aws quickstart metric. This would allow us to visualize/analyze instances where quickstart fails or CloudFormation editor window fails to open.
## Checklist

- [X] I have read the **README** document
- [X] I have read the **CONTRIBUTING** document
- [X] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: telemetry/definitions/commonDefinitions.json [hunk 1]
```diff
@@ -414,7 +415,8 @@
         },
         {
             "name": "cloudformation_open",
-            "description": "Open a CloudFormation stack in the stack viewer"
+            "description": "Open a CloudFormation stack in the stack viewer",
+            "metadata": [ { "type": "result", "required": false  } ]
```
Comment (awschristou) [inline telemetry/definitions/commonDefinitions.json:419]: Other "open" metrics have a required result field. What is the motivation for making these ones optional?

Comment (shruti0085) [reply telemetry/definitions/commonDefinitions.json:419]: I was not certain how this change will affect jetbrains/vscode, so in order to minimize any disruption to them, I made it optional.

Comment (hunterwerlla) [reply telemetry/definitions/commonDefinitions.json:419]: We don't update automatically, if it's an operation that can fail there is no issue with adding result

**Review by awschristou — APPROVED** [review]

File: telemetry/definitions/commonDefinitions.json (hunk 1 above)
Comment (shruti0085) [reply telemetry/definitions/commonDefinitions.json]: Done

**Pull request merged**